### PR TITLE
fix: login in with a second account during a call via deep links (WPB-8937)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-8937" title="WPB-8937" target="_blank">BugWPB-8937</a>  [Android] Deeplink can bypass adding a new account during a call
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

While in a call, users can login to another account via deep links

### Solutions

Show a toast instead of login dialog in case there an ongoing call

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt [hunk 1]
```diff
@@ -255,12 +257,15 @@ class WireActivityViewModel @Inject constructor(
         return intent?.action == Intent.ACTION_SEND || intent?.action == Intent.ACTION_SEND_MULTIPLE
     }
 
+    private suspend fun canLoginThrowDeepLinks() = observeEstablishedCalls().first().isEmpty()
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt]:
```suggestion
    private suspend fun canLoginThroughDeepLinks() = observeEstablishedCalls().first().isEmpty()
```

**Review by alexandreferris — APPROVED** [review]
Nice catch! Just a nitpick on naming

**Comment by AndroidBob** [comment]
Build [4526](https://10.10.124.134/job/AR-build-pipeline/4526/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.8.0(9542376)(9542376)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.8.0(9542376)(9542376)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [4532](https://10.10.124.134/job/AR-build-pipeline/4532/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.8.0(9549432)(9549432)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.8.0(9549432)(9549432)-staging-compat.apk)

**Review by yamilmedina — APPROVED** [review]

**Pull request merged**